app/assets/stylesheets/components/_address.scss

```diff
@@ -0,0 +1,3 @@
+address {
+  white-space: pre;
+}
```

Comment:
Should probably put this in `base/` somewhere as it's styling an element directly: https://github.com/guidance-guarantee-programme/output/blob/master/app/assets/stylesheets/base/_forms.scss